app/helpers/vacancies_helper.rb

```diff
@@ -90,4 +90,17 @@ def vacancy_school_visits_hint(vacancy)
     organisation = organisation_type_basic(vacancy.parent_organisation).tr(" ", "_")
     t("helpers.hint.publishers_job_listing_applying_for_the_job_form.#{organisation}_visits")
   end
+
+  def vacancy_step_completed?(vacancy, step_number)
+    return false if vacancy.completed_step.nil?
+
+    step_number <= vacancy.completed_step
+  end
+
+  def steps_to_display(steps)
+    steps_to_remove = current_organisation.is_a?(School) ? %i[supporting_documents review] : %i[supporting_documents schools review]
+
+    steps.transform_values { |step_details| step_details[:number] - steps_adjust }
```

Comment:
I think steps_adjust should be an argument passed in too, for the same reason as https://github.com/DFE-Digital/teaching-vacancies/pull/3524/files#r639838273